cmd/juju/application/removeunit.go

```diff
@@ -206,6 +219,19 @@ func (c *removeUnitCommand) unitsHaveStorage(unitNames []string) (bool, error) {
 // Run connects to the environment specified on the command line and destroys
 // units therein.
 func (c *removeUnitCommand) Run(ctx *cmd.Context) error {
+	noWaitSet := false
+	forceSet := false
+	c.fs.Visit(func(flag *gnuflag.Flag) {
```

Comment:
This should be done in Init()
Generally, unless there's a good reason, arg checking is done as early as possible.

Reply:
No, it cannot because we need to have a flagset populated to know if the users have provided the options or not.